--- origsrc/liferea-1.4.18/configure.ac	2008-07-27 12:30:49.000000000 -0500
+++ src/liferea-1.4.18/configure.ac	2008-07-27 15:12:36.296875000 -0500
@@ -23,6 +23,7 @@
 AC_PROG_CC
 AM_PROG_CC_STDC
 AM_PROG_CC_C_O
+AC_PROG_CXX
 AC_HEADER_STDC
 AC_PROG_INSTALL
 AC_PROG_LN_S
@@ -354,30 +355,6 @@
 AC_SUBST(PACKAGE_CFLAGS)
 AC_SUBST(PACKAGE_LIBS)
 
-AC_MSG_CHECKING([for some Win32 platform])
-case "$host" in
-  *-*-cygwin*|*-*-mingw*)
-    platform_win32=yes
-    ;;
-  *)
-    platform_win32=no
-    ;;
-esac
-AC_MSG_RESULT([$platform_win32])
-AM_CONDITIONAL(PLATFORM_WIN32, test "$platform_win32" = "yes")
-
-AC_MSG_CHECKING([for native Win32])
-case "$host" in
-  *-*-mingw*)
-    os_win32=yes
-    ;;
-  *)
-    os_win32=no
-    ;;
-esac
-AC_MSG_RESULT([$os_win32])
-AM_CONDITIONAL(OS_WIN32, test "$os_win32" = "yes")
-
 uname=`uname`
 AC_DEFINE_UNQUOTED(OS, $uname, [defines a OS version string, used for OS specific code])
 AC_DEFINE_UNQUOTED(OSNAME, "$uname", [defines a OS version string, used for the user agent string])
--- origsrc/liferea-1.4.18/src/Makefile.am	2007-08-19 09:25:39.000000000 -0500
+++ src/liferea-1.4.18/src/Makefile.am	2008-07-27 15:12:36.296875000 -0500
@@ -8,7 +8,7 @@
 WEBKIT = webkit
 endif
 
-SUBDIRS = net parsers notification ui fl_sources scripting . $(GTKHTML2) $(WEBKIT) mozilla
+SUBDIRS = net parsers ui fl_sources . notification scripting $(GTKHTML2) $(WEBKIT) mozilla
 
 AM_CPPFLAGS = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
@@ -87,3 +87,6 @@
 liferea_bin_LDFLAGS = -Wl,--export-all-symbols,--out-implib,liferea-bin.a
 
 endif
+
+notification/liblinotification.a:
+	(cd notification && $(MAKE) liblinotification.a)
--- origsrc/liferea-1.4.18/src/net/netio.c	2008-03-18 15:38:57.000000000 -0500
+++ src/liferea-1.4.18/src/net/netio.c	2008-07-27 15:12:36.312500000 -0500
@@ -340,7 +340,7 @@
 		snprintf(port_str, sizeof(port_str), "%d", proxyport);
 		realhost = g_strdup(proxyname);
 	}
-	debug2(DEBUG_NET, "host=%s port=%s", realhost, port_str);
+	debug2(DEBUG_NET, "host=%s port=%s", realhost, port);
 	
 	ret = getaddrinfo(realhost, port_str, &hints, &res);
 	
--- origsrc/liferea-1.4.18/src/notification/Makefile.am	2008-01-29 15:11:41.000000000 -0600
+++ src/liferea-1.4.18/src/notification/Makefile.am	2008-07-27 15:12:36.312500000 -0500
@@ -24,4 +24,12 @@
 liblinotiflibnotify_la_LDFLAGS = -avoid-version -module
 
 pkglib_LTLIBRARIES = liblinotiflibnotify.la
+
+if PLATFORM_WIN32
+
+liblinotiflibnotify_la_LDFLAGS += -no-undefined -Wl,$(top_builddir)/src/liferea-bin.a
+liblinotiflibnotify_la_DEPENDENCIES = $(top_builddir)/src/liferea-bin$(EXEEXT)
+
+endif
+
 endif
--- origsrc/liferea-1.4.18/src/scripting/Makefile.am	2007-07-23 12:56:50.000000000 -0500
+++ src/liferea-1.4.18/src/scripting/Makefile.am	2008-07-27 15:12:36.328125000 -0500
@@ -33,6 +33,6 @@
 LIFEREA_WRAP_CFLAGS=$(PACKAGE_CFLAGS:-pthread=)
 
 liferea_wrap.h: liferea.i
-	swig -lua $(LIFEREA_WRAP_CFLAGS) -o liferea_wrap.h liferea.i
+	swig -lua $(LIFEREA_WRAP_CFLAGS) -o $@ $<
 	
 endif
