--- origsrc/liferea-1.4.22b/configure.ac	2008-11-11 01:04:44.620923500 -0600
+++ src/liferea-1.4.22b/configure.ac	2008-11-11 01:41:14.699048500 -0600
@@ -167,6 +167,9 @@
 if test "x$enable_gecko" = "xyes"; then
 
 GECKO_INIT([GECKO])
+
+fi
+
 GECKO_DEFINES
 
 AC_SUBST([GECKO])
@@ -181,6 +184,7 @@
 xulrunner) gecko_min_version=1.8; is_xulrunne=yes ;;
 libxul*) gecko_min_version=1.9; is_xulrunner=yes ;;
 *firefox) gecko_min_version=1.5; is_mozilla=yes ;;
+'') ;;
 *) AC_MSG_ERROR([$gecko_cv_gecko is not supported])
 esac
 
@@ -194,8 +198,6 @@
 if test "x$is_mozilla" = "xyes"; then
 	PKG_CHECK_MODULES([MOZILLA],[${gecko_cv_gecko}-xpcom >= $gecko_min_version $gecko_cv_extra_pkg_dependencies])
 fi
-
-fi
  
 dnl ******
 dnl WebKit
