--- origsrc/liferea-1.8.1/configure.ac	2012-03-03 15:04:53.000000000 -0600
+++ src/liferea-1.8.1/configure.ac	2012-03-05 17:36:59.821001600 -0600
@@ -125,7 +125,6 @@ src/webkit/Makefile
 src/parsers/Makefile
 src/fl_sources/Makefile
 src/notification/Makefile
-src/sync/Makefile
 src/ui/Makefile
 doc/Makefile
 doc/html/Makefile
--- origsrc/liferea-1.8.1/src/main.c	2012-03-03 15:04:53.000000000 -0600
+++ src/liferea-1.8.1/src/main.c	2012-03-05 17:45:46.147303600 -0600
@@ -47,7 +47,6 @@
 #include "xml.h"
 #include "ui/ui_session.h"
 #include "ui/liferea_shell.h"
-#include "sync/avahi_publisher.h"
 #include "notification/notification.h"
 
 static enum {
